src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -4885,6 +5049,49 @@ components:
           example: '2018-12-15T00:00:00.001Z'
           minLength: 24
           maxLength: 24
+    InitialServerConf: # ok
+      type: object
+      description: security server initial configuration
+      required:
+        - member_class
+        - member_code
+        - security_server_code
+        - software_token_pin
+        - ignore_warnings
```

Comment:
Maybe should not be required (also default value is meaningless if it is), other endpoints do not require ignore_warnings parameters.

Reply:
Due to the partial init refactoring there are not required parameters anymore in this action (including the `ignore_warnings` flag)